wremnants/muon_efficiencies.py

```diff
@@ -9,7 +9,7 @@
 
 data_dir = f"{pathlib.Path(__file__).parent}/data/"
 
-def make_muon_efficiency_helpers(filename = data_dir + "/testMuonSF/scaleFactorProduct_28Oct2021_nodz_dxybs_genMatchDR01.root", era = None, is_w_like = False, max_pt = np.inf):
+def make_muon_efficiency_helpers(filename = data_dir + "/testMuonSF/scaleFactorProduct_29Sept2022_vertexWeight_OSchargeExceptTracking.root", era = None, is_w_like = False, max_pt = np.inf):
```

Comment:
Why does it need test in the name? ;)

Reply:
It is just historical when the efficiencies were changing every week, the idea was to have them in a test folder and keep the nominal file elsewhere. We can change the folder name

Reply:
Well MuonSF would be fine imo, the ROOT file name is already pretty descriptive

Reply:
Keeping it for now, we can polish later